growth-ring/src/wal.rs

```diff
@@ -1301,3 +1314,25 @@ impl WalLoader {
 }
 
 pub const CRC32: crc::Crc<u32> = crc::Crc::<u32>::new(&crc::CRC_32_CKSUM);
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use test_case::test_case;
+
+    #[test_case("foo", Err("not a log file"); "no log extension")]
+    #[test_case("foo.log", Err("not a log file"); "invalid digit found in string")]
+    #[test_case("0000001.log", Ok(1); "happy path")]
+    #[test_case("1.log", Ok(1); "no leading zeroes")]
+
+    fn test_get_fid(input: &str, expected: Result<u64, &str>) {
```

Comment:
Nit: prefixing test functions with `test_` is a stutter, since they are already in the `tests` module.